sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationClientBuilder.java

```diff
@@ -266,14 +272,27 @@ public TextTranslationClientBuilder credential(KeyCredential credential) {
      *
      * @param region where the Translator resource is created.
      * @return The updated {@link TextTranslationClientBuilder} object.
-     * @throws NullPointerException If {@code tokenCredential} is null.
+     * @throws NullPointerException If {@code region} is null.
      */
     public TextTranslationClientBuilder region(String region) {
         Objects.requireNonNull(region, "'region' cannot be null.");
         this.region = region;
         return this;
     }
 
+    /**
+     * Sets the Azure Resource Id used to authorize requests sent to the service.
+     *
+     * @param azureResourceId Id of the Translator Resource.
+     * @return The updated {@link TextTranslationClientBuilder} object.
+     * @throws NullPointerException If {@code azureResourceId} is null.
+     */
+    public TextTranslationClientBuilder azureResourceId(String azureResourceId) {
```

Comment:
We should just name this method `resourceId`.

Reply:
Done